shared/src/main/scala/Elements.scala

```diff
@@ -623,6 +623,24 @@ object Elements:
         MiscHelpers.reduce(a, b)
     }
 
+    val sort: Monad = addElem(
+      Monad,
+      "S",
+      "Sort ascending",
+      List(
+        "sort",
+        "sortasc",
+        "sort-asc",
+      ),
+      "a: any -> convert to list and sort ascending",
+    ) {
+      // should do something else for num overload later
+      case (a: VAny) =>
```

Comment:
`(a: VAny)` can be replaced with `a`, since we only deal with `VAny`s, and since that would become `case a =>`, we don't even need pattern matching, so `addFull` can be used and this line can become `a =>`